src/main/java/org/bytedeco/javacv/Frame.java

```diff
@@ -71,11 +71,12 @@
             DEPTH_LONG   = -64,
             DEPTH_FLOAT  =  32,
             DEPTH_DOUBLE =  64;
-    
+
     /** Constants defining data type in the frame*/
     public static enum Type {
         VIDEO,
         AUDIO,
+        OTHER
```

Comment:
Let's consider this type of frame as `DATA` instead of `OTHER`.

Reply:
Will change the enum to `DATA`